stmhal/boards/STM32F429DISC/pins.csv

```diff
@@ -0,0 +1,38 @@
+SW,PA0
+PA0,PA0
+PA5,PA5
+PA8,PA8
+PA9,PA9
+PA10,PA10
+PA13,PA13
+PA14,PA14
```

Comment:
I was no sure which kind of pins should be included here. As I excluded the pins which are hard wired to certain periphery:
* PA0: Switch
* PA1: MEMS_INT1
* PA2: MEMS_INT2
* PA3: LCD_B5
* PA4: LCD_VSYNC
* PA6: LCD_G2
* PA7: ACP_RST is on a connector => has to be included in list
* PA11: LCD_R4
* PA12: LCD_R5
* PA15: TP_INT2 on MEMS
* PB0: LCD_R3
* PB1: LCD_R6
* PB5: SDRAM SDCKE1
* PB6: SDRAM SDNEI
* PB8: LCD_B6
* PB9: LCD_B7
* PB10: LCD_G4
* PB11: LCD_G5
* PB12: USB_OTG_FS_ID
* PB13: USB VBUS_FS
* PB14: USB OTG_FS_DM
* PB15: USB OTG_FS_DP
* PC0: SDRAM SDNWE
* PC1: SDRAM NCS_MEMS_SPI
* PC2: LCD CSX
* PC4: USB OTG_FS_PSO
* PC5: USB OTG_FS_OC (Switch 5V to USB bus)
* PC6: LCD_HSYNC
* PC7: LCD_G6
* PC10: LCD_R2
* PD0/1: SDRAM _D2/3
* PD2: Is included in pins.csv SB25 is not fitted
* PD3:LCD_G7
* PD4:  Is included in pins.csv SB24 is not fitted
* PD5:  Is included in pins.csv SB23 is not fitted
* PD6: LCD_B2
* PD7:  Is included in pins.csv SB22 is not fitted
* PD8-10: SDRAM D13-15
* PD11-13: LCD
* PD14/15: SDRAM D0/1
* PE0/1: SDRAM NBL0/1
* PE7-PE15: SDRAM D4-D12
* PF0-PF5: SDRAM A0-A5
* PF7-9: SPI5, will be included in pins.csv
* PF10: LCD Enable
* PF11:SDRAM SDNRAS
* PF12-PF15: SDRAM A6-A9
* PG0-PG1:  SDRAM A10-A11
* PG4/5: SDRAM BA0/BA1
* PG6: LCD_R7
* PG7: LCD_CLK
* PG8: SDRAM SDCLK
* PG10: LCD_G3 (Wrong connected? Alternate function LCD_B2)
* PG11: LCD_B3
* PG12: LCD_B4 (Wrong connected? Alternate function LCD_B1)
* PG15: SDRAM SDNCAS